src/main/java/org/craftercms/studio/impl/v2/utils/git/GitUtils.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (C) 2007-2022 Crafter Software Corporation. All Rights Reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as published by
+ * the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.craftercms.studio.impl.v2.utils;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.apache.commons.io.FileUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Utility methods for Git operations.
+ *
+ * @author Phil Nguyen
+ */
+
+public class GitUtils {
```

Comment:
it would be a good experience for @phuongnq 

@phuongnq please move the Git Utils to the `commons` project and use them in both Deployer and Studio.

Reply:
Moved to `commons`. The following PR must be checked and merged first: https://github.com/craftercms/commons/pull/303